src/scss/grommet-core/_objects.paragraph.scss

```diff
@@ -1,6 +1,6 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
-.grommet .#{$grommet-namespace}paragraph {
+p.#{$grommet-namespace}paragraph {
```

Comment:
These styles get applied when I remove `.grommet` from the selector.
Specificity level here is 11. Just having `.grommetux-paragraph` without `.grommet` has a specificity of 10.

![image](https://cloud.githubusercontent.com/assets/3210082/18300236/009838b0-7463-11e6-95a6-7f96b6672ca0.png)

And having `.grommet` before the selector is making it too specific. 

![image](https://cloud.githubusercontent.com/assets/3210082/18300182/98fe4f5a-7462-11e6-8447-251816282e37.png)